blogs/whats_new_1.6/index.md

```diff
@@ -0,0 +1,225 @@
+
+Now this will only deploy on nodes with the `ebpf-agent=true` label.  Set this label by replacing *node-name* with your node name below.
+
+```
+oc label nodes node-name ebpf-agent=true
+```
+
+### Filter at eBPF level
+
+Network Observability always had great filtering capability in the **Network Traffic** panel.  However, that is done at the UI level, meaning the flows are still generated and stored, and hence resources are being used.  If resources are a major concern, consider this feature ***if*** the filtering fits your requirements.
+
+If you want to generate flows (or not generate flows) for very specific data, you can specify a single filter to do this.  Here's an example to only generate flows for network traffic on TCP port 6443.
+
+```
+  agent:
+    ebpf:
+      flowFilter:
+        enable: true
+        action: Accept
+        cidr: 0.0.0.0/0  # refers to any traffic
+        ports: 6443
+        protocol: TCP
+```
+
+Enter `oc edit flowcollector -n netobserv-privileged`.  Again, look for the `ebpf` section and add the rest of the lines starting at `flowFilter`.  The `cidr` specifies a network address and prefix.  The example refers to any traffic, but for a specific network, use something like 10.0.62.0/24.  The feature is limited to one filter and many of the attributes can only have one value.
+
+### eBPF Agent metrics
+
+You can view statistics for eBPF Agent.  This is in Web console under **Observe > Dashboards**.  Select **NetObserv / Health** in the Dashboard dropdown (Figure 5).  There are graphs for eviction rates, dropped flow rates, and more.
+
+![eBPF Agent statistics](images/ebpf_agent_statistics.png)
+_<div style="text-align: center">Figure 5: eBPF Agent statistics</div>_
+
+### Other eBPF changes
+
+Here are a few other minor eBPF changes.
+
+1. Option to alert if it's dropping flows<br>
+There is a new option in FlowCollector to control whether an alert is shown if it's dropping flows (Figure 6).  By default, this is enabled.
+
+![Disable alerts](images/ebpf_metrics_configuration.png)
+_<div style="text-align: center">Figure 6: eBPF Agent configuration - Disable alerts</div>_
+
+2. Use enhanced TCx hook<br>
+Internally, the eBPF Agent uses the Traffic Control (TC) hook to probe the ingress and egress traffic on the interfaces.  In OCP 4.16, which upgrades RHEL to 9.4, it can leverage the enhanced TCx hook for performance gains.
+
+3. Always save DNS flags<br>
+If the *DNSTracking* feature is enabled, regardless of the sampling setting, it always saves the DNS id and flags.  Previously, it would discard this data if the flow didn't exist.
```

Comment:
It's not exactly this. It's that, to compute DNS latency, you need to catch the request and the response - it this is quite unlikely to happen as soon as you have sampling >1, you need some luck to capture both the request and the response. Previously, it was all or nothing: we were either writing all the DNS stats, or none, tying error codes & id with the availability of the latency. Now we still write codes & id event if we don't have the latency. It's still tied to the sampling config, ie. unsampled flows won't get their DNS stats, but you don't need anymore to have both the request and the response sampled.

Reply:
```suggestion
Previously, with the *DNSTracking* feature enabled, it was unlikely to generate DNS statistics as soon as sampling was configured greater than 1, because the probe was expecting to catch both the request and the response. This had been changed so that it always saves the DNS id and flags. Only the DNS latency remains unlikely to have with high sampling configured.
```